itou/www/apply/views/list_views.py

```diff
@@ -120,6 +124,35 @@ def list_for_prescriber_exports_download(request, month_identifier=None):
     return stream_xlsx_export(job_applications, filename)
 
 
+def _add_administrative_criteria(job_applications):
+    diagnoses_ids = tuple(
+        job_application.jobseeker_eligibility_diagnosis
+        for job_application in job_applications
+        if job_application.jobseeker_eligibility_diagnosis is not None
+    )
+
+    diagnosis_criteria = defaultdict(list)
+    for selected_criteria in (
+        SelectedAdministrativeCriteria.objects.filter(eligibility_diagnosis__in=diagnoses_ids)
+        .select_related("administrative_criteria")
+        .order_by("administrative_criteria__level", "administrative_criteria__name")
+    ):
+        diagnosis_criteria[selected_criteria.eligibility_diagnosis_id].append(
+            selected_criteria.administrative_criteria
+        )
+
+    for job_application in job_applications:
+        ja_criteria = diagnosis_criteria[job_application.jobseeker_eligibility_diagnosis]
+        if len(ja_criteria) > 4:
```

Comment:
On ne risque pas de masquer un critère s’il y en a exactement 4 ?
```suggestion
        if len(ja_criteria) >= 4:
```

Reply:
S'il y en a 4, on les affiche tous (cf test), mais à partir de 5 on en affiche que 3 (plus une ligne "+ 2 autres critères")

Reply:
Malin !